time-machine/README.md

```diff
@@ -0,0 +1,97 @@
+# Time Machine Sample Application
+
+## Overview
+
+Organizations frequently report HR metrics to external stakeholders and use them for internal scorecarding.
+However, subsequent corrections to historical data can introduce discrepancies between the initially reported values and
+the current ones.
+
+This project addresses the challenge of maintaining data integrity for accurate historical reporting and analysis.
+It provides a solution to preserve original metric values, ensuring consistency and avoiding issues caused
+by data corrections at a specific point in time.
+
+## Solution
+
+The sample application comprises two steps:
+
+1. Downloading the current data state:
+    - Use the Data Query API to download the current data state and save it in a temporary CSV file.
+    - An aggregate query stored in a JSON file is required for this step. A sample query is located in
+      [./queries/gender-resignation-joblevel.json](queries%2Fgender-resignation-joblevel.json).
+
+2. Uploading this csv file using Data Upload API:
+    - Use the Data Upload API to upload the CSV file.
+    - The Visier platform maps this data to a pre-created Overlay, which will be used to access data as it was at a
+      specific point in time.
+
+To process the uploaded file, the Visier Platform requires the creation of a Source, Overlay, and Mapping.
+Here are the instructions:
+
+1. [Source creation](https://docs.visier.com/developer/Studio/data/sources/source-create.htm)
+2. [Overlay creation](https://docs.visier.com/developer/Analytic%20Model/analytic-objects/overlays/overlays-configure.htm)
+3. [Mapping from Source to Overlay](https://docs.visier.com/developer/Studio/data/mappings/mapping-add.htm)
+
+Instead of manually creating the source, overlay, and mapping, you can import these pre-configured components from the
+provided files:
+
+1. [./import/WFF_d3m_source.zip](import%2FWFF_d3m_source.zip)- to import source.
```

Comment:
Will this import work on tenants named something other than `WFF_d3m`?

Reply:
It worked, but there should be manually assigned Data Category and Tenant Settings.
I changed archives to have 3 parts:
1. source
2. data_category, tenant_settings
3. overlay and mapping